hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/error/ErrorBuilder.java

```diff
@@ -10,78 +12,204 @@
 	
 	private static String ack= "ACK";
 	
-	private static String version = "2.4\n";
+	private static String msh= "MSH";
```

Comment:
I believe all the Strings (expect sendingFacility and receivingFacility which I will mention separately) should be static final and follow CONSTANT naming conventions (see https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)